### ISSUE TITLE
post delete, zip archive is corrupted

### ISSUE THREAD

**Issue by soumyart** [gregnb/filemanager-webpack-plugin#23]
## Config
```js
new FileManagerPlugin({
      'onEnd': [
        {
          'copy': [
            {'source': './docs/file.min.js', 'destination': './dist/'},
            {'source': './docs/file.min.js.map', 'destination': './dist/'},
            {'source': './docs/file.css', 'destination': './dist/'},
            {'source': './docs/fonts', 'destination': './dist/fonts'},
          ]
        },
        {
          'archive': [
            {'source': './dist', 'destination': './file.zip'},
          ]
        },
        {
          'move': [
            {'source': './file.zip', 'destination': './dist/file.zip'}
          ]
        },
        {
          'delete': [
            './dist/file.min.js.map',
            './dist/file.min.js',
            './dist/file.css',
            './dist/fonts',
          ]
        }
      ]
    })
```

## Issue
After creating the zip file I'm cleaning up the dist directory. But post build competed, zip file is corrupted and the size of zip is also reduced.
If I remove the `delete` option then there is no issue.

## Your Environment

| Tech         | Version |
|--------------|---------|
| filemanager-plugin-webpack  |1.0.24|
| node        | v8.9.3 |
| OS      |  Windows 8.1 Enterprise |

**Comment by gregnb** [comment]
Ok, thank you I will look into this issue

**Comment by gregnb** [comment]
This has been fixed in version 1.0.25
